pkgs/sops-install-secrets/main.go

```diff
@@ -870,6 +897,21 @@ func installSecrets(args []string) error {
 		return err
 	}
 
+	if manifest.UserMode {
+		rundir, ok := os.LookupEnv("XDG_RUNTIME_DIR")
+		if !ok {
+			return fmt.Errorf("$XDG_RUNTIME_DIR is not set!")
```

Comment:
I'm building my home-manager config with root as part of my nixos-config. `Nixos-rebuild switch` fails for me with:
```
error: builder for '/nix/store/bhscqh6gvv32cda919mkh2sa7yaa9chi-manifest.json.drv' failed with exit code 1;
       last 1 log lines:
       > /nix/store/ppl4vbv67c420zk9r87izdsfwms5q2ja-sops-install-secrets-0.0.1/bin/sops-install-secrets: $XDG_RUNTIME_DIR is not set!
       For full logs, run 'nix log /nix/store/bhscqh6gvv32cda919mkh2sa7yaa9chi-manifest.json.drv'.
error: 1 dependencies of derivation '/nix/store/a93kndskwslvnaaymqqk4svjhj7s33gg-sops-nix-user.drv' failed to build
```
I suspect this happens during the check phase. It may be sufficient to just ignore `$XDG_RUNTIME_DIR` during `--check-mode`.